archethic/config.js

```diff
@@ -29,7 +29,10 @@ export default {
       },
     },
     sepolia_ethereum: {
-      endpoint: "https://sepolia.infura.io/v3/3a7a2dbdbec046a4961550ddf8c7d78a",
+      endpoints: [
+        "https://sepolia.infura.io/v3/3a7a2dbdbec046a4961550ddf8c7d78a",
+        "https://eth-sepolia.g.alchemy.com/v2/L4TxVWLSkKKzP2njMgq_smU6Fc_lpZ0S",
```

Comment:
You should use Archethic's API keys
```suggestion
        "https://eth-sepolia.g.alchemy.com/v2/eLhVAxz79HO5n2y98mdIl_gMkKSDc3G8",
```